lib/reader.js

```diff
@@ -1,115 +1,53 @@
 'use strict';
 
-const JSONStream = require('JSONStream');
 const pack = require('browser-pack');
-const mergeStream = require('merge-stream');
-const stream = require('readable-stream');
-
-// Ensure output order is the same as input order. The sorter only sorts the
-// modules, not execution order. `order` is consumed by browserify's
-// `browser-pack`. The feeds already have an order, but they are always `0` as
-// each feed only have a single entrypoint.
-class OrderDecorator extends stream.Transform {
-    constructor (order) {
-        super({
-            objectMode: true,
-        });
-
-        this.order = order;
-    }
-
-    _transform (row, encoding, next) {
-        if (row.entry === true) {
-            row.order = this.order;
-        }
-        next(null, row);
-    }
+const getStream = require('get-stream');
+const assert = require('assert');
+
+function getRows (feeds, existMap = new Map()) {
+    return feeds
+        .reduce((acc, feed, order) => {
+            feed.forEach((row) => {
+                // add browserify.order and flatten
+                delete row.order;
+                if (row.entry === true) {
+                    row.order = order;
+                }
+                acc.push(row);
```

Comment:
so much mutation in this `reduce` hurts 😅

Reply:
I know. Real bad.